src/group_theory/quotient_group.lean

```diff
@@ -332,7 +332,13 @@ lift _ ((mk' _).comp φ') $
   λ g ⟨h, (hg : h ^ n = g)⟩, (eq_one_iff _).mpr ⟨_, by simpa only [← hg, map_zpow]⟩
 
 @[to_additive, simp]
-lemma hom_quotient_zpow_of_hom_right_inverse (h : function.right_inverse ψ' φ') (n : ℤ) :
+lemma hom_quotient_zpow_of_hom_comp (n : ℤ) :
+  (hom_quotient_zpow_of_hom φ' n).comp (hom_quotient_zpow_of_hom ψ' n)
+    = hom_quotient_zpow_of_hom (φ'.comp ψ') n :=
+monoid_hom_ext _ rfl
+
+@[to_additive, simp]
+lemma hom_quotient_zpow_of_hom_comp_of_right_inverse (h : function.right_inverse ψ' φ') (n : ℤ) :
```

Comment:
I'd argue this lemma isn't that useful any more

Reply:
The following definition becomes really ugly without this?